Marlin/ultralcd.cpp

```diff
@@ -1523,6 +1503,59 @@ void kill_screen(const char* lcd_msg) {
     lcd_return_to_status();
   }
 
+  #if ENABLED(FILAMENT_LOADUNLOAD_SUPPORT)
+    void lcd_filament_load_unload_menu() {
+      START_MENU();
+      MENU_BACK(MSG_PREPARE);
+      // Load filament
+      #if E_STEPPERS == 1
+        if (!thermalManager.targetTooColdToExtrude(active_extruder))
+          MENU_ITEM(gcode, MSG_FILAMENTLOAD, PSTR("M701"));
+      #else
+        if (!thermalManager.targetTooColdToExtrude(0))
+          MENU_ITEM(gcode, MSG_FILAMENTLOAD1, PSTR("M701 T0"));
```

Comment:
Please use `MSG_FILAMENT_LOAD MSG_E0` here. Then there is no need for multiple redundant `MSG_FILAMENTLOADn` defines.